Certification standard/main.html

```diff
@@ -0,0 +1,221 @@
+        How is the content agreed on and validated?
+        ?
+        ]
+    </section>
+
+    <section id="introduction" class="informative">
+      <h2>Introduction</h2>
+
+      This document is XXX.
+
+
+      [Todo:
+      - What is certification? What certification is NOT?
+      - Who could/should be seeking certifications?
+      - How to contribute? How to add vulnerabiities?
+      - Process for new versions?
+      - Define the syntax and semantics of the requirements/certification objectives
+      - Laws and other applicable standards? GDPR? MiCA?
+      ]
+
+    </section>
+
+    <section id='ccwg-conformance'>
+      <h2> Certification process</h2> 
+
+      <section>
+        <h3> What is certified? </h3>
+
+        A certification refers to a DApp, which means the source code of the on-chain component of a smart contract or several smart contracts, along the UPLC compiled from the source code and a given parametrisation.
+
+        [Discussion: Do we want to certify Dapps with general parameters or only instantions of such Dapps?]
+      </section>
+      <section>
+        <h3> On-chain certificates </h3>
+        
+        Say something about our CIP on certification metadata.
+      </section>
+      <section>
+        <h3>Who can issue certificates ? </h3>
+
+        This section XXX
+        
+        [Todo:
+        - Do we keep a list of trusted issuer? Can anyone issue certificates?
+        - How do we join the list?
+        - How do we remove someone from the list?
+        ]
+      </section>
+    </section>
+
+    <section id='certlevels'>
+      <h2>Certification Levels</h2>
+
+      Three certification levels are defined, named 1, 2, and 3. [TBC: We have a chance to rename, redefine, reorganize, renumber the levels, etc.]
+
+      <section id='certlevels-l1'>
+        <h3>Level 1: Automated testing</h3> 
+
+        Certification at level 1 is intended for verification of a <a>DApp</a> that can be performed by an unguided automated tool. The tool shall be able to determine if the code meets its requirements and is not vulnerable to some vulnerabilities [Discussion: Should we relate to the list of vulnerabilities and define the ones that can/should be dealt at this level, because the tools can do it].
+
+        [Todo (or not but tbd):
+        - Vulnerabilities to be checked
+        - Metrics to be reached
+        - Tracability to documentation
+        - List of tools/techniques? Fuzzing, mutation testing, concolic testing, symbolic execution, property based testing, etc.?
+        - Tool qualification
+        - [Others?]
+        ]
+      </section>
+
+      <section id='certlevels-l2'>
+        <h3>Level 2: Manual audit</h3>
+
+        Certification at level 2 is intended for a <a>DApp</a> that has been carefully reviewed by an auditor, through a manual analysis with possibly the use of automated tools. [Discussion: How de we split this from auditor doing only the work of level 1?]
```

Comment:
re Discussion: one distinction between an assessor doing only level 1 and doing level 2 would probably be _assessing the quality of the developer-delivered tests, models etc_ emphasising the earlier comment on line 84.